File: Python/mario_more.py
import cs50

#check user input
while True:
    h = cs50.get_int("Height: ")
    if h > 0 and h <= 8:break

#prints the # according to the size specified by the user 'h'
for i in range(h):
    for j in range(h):
        if i + j < h - 1:
            print(" ",end='')
        else:
            print("#",end='')
    print("  ",end='')
    #range(start, end, step)
    for j in range(h, 0, -1):
        if j + i > h - 1:
            print("#",end='')
        else:
            print(" ",end='')
    print()

File: C/mario_more.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{   //check user input
    do
    {
       int h = get_int ("Height: ");
    }
    while (h<1 || h>8);
    //#prints the # according to the size specified by the user 'h'
    for (int n=0; n<h; n++)
    {
        for (int j=0; j<h; j++)
        {
            if (n + j < h - 1)
                printf(" ");
            else
                printf("#");
        }
        printf("  ");
        for (j=h; j>0; j--)
        {
            if (j+n > h - 1)
                printf("#");
            else
                printf(" ");
        }
        printf("\n");
    }
}
